h2gis-functions/src/test/java/org/h2gis/functions/spatial/ogc/OGCSpatialFunctionTest.java

```diff
@@ -632,4 +632,42 @@ public void test_ST_LengthOnPolygon() throws SQLException {
         assertFalse(rs.next());
         rs.close();
     }
+
+    @Test
+    public void test_ST_DistanceSpherePointToPoint() throws Exception {
+        Statement st = connection.createStatement();
+        ResultSet rs = st.executeQuery("SELECT ST_DistanceSphere(buildings.position, bridges.position) " +
+            "FROM buildings, bridges " +
+            "WHERE buildings.address = '215 Main Street' AND bridges.name = 'Cam Bridge'");
+        assertTrue(rs.next());
+        assertEquals(1896135.5064393785, rs.getDouble(1),1e-9);
+    }
+
+    @Test
+    public void test_ST_DistanceSpherePointToPolygon() throws Exception {
+        Statement st = connection.createStatement();
+        ResultSet rs = st.executeQuery("SELECT ST_DistanceSphere(position, boundary) FROM bridges, named_places " +
```

Comment:
Could please use a more generic query for all tests as ST_DistanceSphere('POINT(0,0)'::GEOMETRY, 'POINT(-118 38)'::GEOMETRY)